README.md

```diff
@@ -0,0 +1,158 @@
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [BUDGET APP] <a name="about-project"></a>
+
+**[Budget app]** is an application designed for users to track their expenditures.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- <a href="https://www.ruby-lang.org/es/">ROR</a>
+- <a href="https://www.postgresql.org/">Postgresql</a>
+- <a href="https://rspec.info/">Rspec</a>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Users can view their transaction and Categories]**
+- **[Users can create a transaction]**
+- **[Users can create a category.]**
+- **[Limit access to web app resources based on authorization rules.]**
+
+- Here is a [Live Demo Link](https://budget-n7m7.onrender.com/users/sign_in)
+- Here is a [presentation video link](https://https://www.loom.com/share/7a884207eb344ed3bae6e2190f7e7d28)
+
```

Comment:
- [ ] It would be nice if you followed all the requirements of your project and show us the piece of code which you are very proud of 👍 to make your video more descriptive and professional 😊
- [ ] Kindly fix link as well because it is broken 😨 It has two HTTPS at the start of the link kindly remove one 👍